pkgs/by-name/ex/expresslrs-configurator/package.nix

```diff
@@ -0,0 +1,101 @@
+{
+  stdenv,
+  fetchzip,
+  autoPatchelfHook,
+  lib,
+  makeWrapper,
+  electron_27,
```

Comment:
Electron 27 is marked as insecure. Is the program not compatible with newer versions?

Reply:
I now upgraded to Electron 33 but it is throwing the same error with the missing devices folder. I new to nix packaging and I am unsure how to investigate this error further. Thank you for your help!